resources/scss/_global.scss

```diff
@@ -282,15 +282,77 @@ progress {
 // Top bar: Logo, Search, Navigation, Login
 
 .top-bar {
+  top: 0;
   background-color: $purple-light;
   position: fixed;
   width: 100%;
   z-index: 10;
   a:hover {
     text-decoration: none;
   }
+  height: 74px;
 }
 
+#spacer {
+  height: 86px;
+}
+
+#filter-bar {
+  height: 80px;
+  margin-top: 74px;
```

Comment:
I think we can add a smaller `font-size` here (maybe 0.8em) and decrease the height (maybe 70px)

Reply:
That looked a little small, I'm doing .85em, 70px. Lmk what you think